ruby/lib/simple_sauce/options.rb

```diff
@@ -43,6 +43,14 @@ def capabilities
     end
     alias as_json capabilities
 
+    def set_capabilities(opts)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not prefix writer method names with `set_`.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4747832)